expr/func_aggregatewithwildcards_test.go

```diff
@@ -0,0 +1,130 @@
+package expr
+
+import (
+	"testing"
+
+	"github.com/grafana/metrictank/api/models"
+)
+
+func TestAggregateWithWildcardsZero(t *testing.T) {
+	f := makeAggregateWithWildcards("sum", []models.Series{}, 0, []expr{{etype: etInt, int: 1}})
+	got, err := f.Exec(make(map[Req][]models.Series))
+	if err := equalOutput([]models.Series{}, got, nil, err); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestAggregateWithWildcardsIdentity(t *testing.T) {
+	testAggregateWithWildcards(
+		"identity",
+		"average",
+		[]models.Series{
+			getSeriesNamed("single", a),
+		},
+		getSeriesNamed("averageSeries(single)", a),
```

Comment:
graphite doesn't add the wrapping "averageSeries(...)" when using aggregateWithWildcards, so I think neither should we